cmd/shadowsocks-httpget/httpget.go

```diff
@@ -14,7 +14,7 @@ import (
 	"strings"
 	"time"
 
-	ss "github.com/shadowsocks/shadowsocks-go/shadowsocks"
```

Comment:
I don't want this be depend on. cause the shadowsocks-go should provide a function full server and client and the MOST IMPORTANT is a easy use lib for other users

Reply:
about locally support AEAD, I have do this in my PR. But it will not merge yet.

Reply:
I don't want that neither. it is a mistake, I didn't noticed that line before. about AEAD, I have to do some jobs to make cow keep working for me, cause my ss provider had upgraded ss cipher to chacha20-ietf-poly1305. according to shadowsocks-go original functions' structure, I couldn't found another better way to support it. maybe yours is better, for now my version is working quite well. btw, maybe this pr should be abandoned, I would make a new pr in the next few days

Reply:
THX for your contribute and maybe we can get another way to contribute

Reply:
parts of this branch implement for dealing Stream and AEAD cipher might not call elegant, maybe even ugly, I hate it a lot. but I am not familar with golang, maybe someone would fixed or rewrote that in future.